src/frontend/src/flows/addDevice/welcomeView/registerTentativeDevice.ts

```diff
@@ -41,6 +44,12 @@ export const registerTentativeDevice = async (
   // Kick-off fetching "ua-parser-js";
   const uaParser = loadUAParser();
 
+  const deviceTrusted = await promptDeviceTrusted({ userNumber });
```

Comment:
Not in this PR, but it would be good to rename this function to `registerNewTentativeDevice` if it's only for new devices.

Reply:
"tentative" devices are always new. There is no existing tentative device.

Reply:
Ahh, ok, I didn't know about it. Thanks.